ansible/playbook.yaml

```
@@ -0,0 +1,94 @@
+- name: Update web servers
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: trello_api_key
+      private: true
+      prompt: "Trello API key"
+
+    - name: trello_api_secret
+      private: true
+      prompt: "Trello API secret"
+
+    - name: trello_api_token
+      private: true
+      prompt: "Trello API token"
+
+# Install dependencies
+  tasks:
+    - name: Install git
+      yum:
+        name:
+          - git
+        state: present
+      become: true
+
+    - name: Install Python 3
+      yum:
+        name:
+          - python3
+        state: present
+      become: true
+
+    - name: Install pip
+      yum:
+        name: python3-pip
+        state: present
+      become: true
```

Not important, but worth noting that we can combine these if we prefer the brevity
```suggestion
    - name: Install git, Python 3 & Pip
      yum:
        name:
          - git
          - python3
          - python3-pip
        state: present
      become: true
```